src/hotrod/api/CountersImpl.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * RemoteCountersImpl.h
+ *
+ *  Created on: May 4, 2018
+ *      Author: rigazilla
+ */
+
+#ifndef SRC_HOTROD_API_COUNTERSIMPL_H_
+#define SRC_HOTROD_API_COUNTERSIMPL_H_
+
+#include "infinispan/hotrod/CounterConfiguration.h"
+#include "hotrod/impl/RemoteCounterManagerImpl.h"
+#include <string>
+#include <future>
+
+namespace infinispan {
+namespace hotrod {
+
+class BaseCounterImpl: public virtual Counter {
+public:
+    BaseCounterImpl(RemoteCounterManagerImpl& rcm, std::string name, CounterConfiguration configuration) :
+            rcm(rcm), name(name), configuration(configuration), removed(false) {
+    }
+
+    std::string getName() {
+        return name;
+    }
+    CounterConfiguration getConfiguration() {
+        return configuration;
+    }
+
+    void setRemoved()
+    {
```

Comment:
The placement of the `{` is inconsistent. In some places it is on the same line as the method declaration, and in some places it is alone on a line. (I personally prefer the first!)